src/Constants/display.js

```diff
@@ -0,0 +1,12 @@
+export const InputComment = Object.freeze({
+  BUY_PRICE: '구입금액을 입력해 주세요.\n',
+  BASIC_NUMBERS: '당첨 번호를 입력해 주세요.\n',
+  BONUS_NUMBER: '보너스 번호를 입력해 주세요.\n',
+  Seperator: { NAME: '쉼표', VALUE: ',' },
+});
+
+export const OutputComment = Object.freeze({
+  STATISTIC_TITLE: '당첨 통계',
+  LINE: '---',
+  SEPERATOR: ', ',
+});
```

Comment:
```suggestion
export const InputComment = Object.freeze({
  BUY_PRICE: '구입금액을 입력해 주세요.\n',
  BASIC_NUMBERS: '당첨 번호를 입력해 주세요.\n',
  BONUS_NUMBER: '보너스 번호를 입력해 주세요.\n',
  SEPERATOR: { NAME: '쉼표', VALUE: ',' },
});

export const OutputComment = Object.freeze({
  STATISTIC_TITLE: '당첨 통계',
  LINE: '---',
  SEPERATOR: { NAME: '쉼표 공백', VALUE: ', ' },
});
```
개인적으로는 일관성있게 코드를 작성하려면 이렇게 바뀌어야 할 것 같은데 민제님 생각은 어떠신가요??

바뀐점
- `Seperator`, `SEPERATOR` 키 이름이 달라서 대문자로 통일
- 위에 `SEPERATOR`는 객체 형식인데 아래는 `value` 값만 들어가서 객체 형식으로 통일

Reply:
통일성은 생각하지 못하고, 실제로 사용하는 값인가? 생각해서 Input과 Output의 Seperator 형태가 달랐던 것 같습니다. (Input은 쉼표를 출력부에 썼는데, Output은 사용하지 않음)좋은 피드백 감사합니다! 😄